error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StringJoin.java

```diff
@@ -40,7 +40,7 @@
 @AutoService(BugChecker.class)
 @BugPattern(
     summary = "Prefer `String#join` over `String#format`",
-    linkType = NONE,
+    link = BUG_PATTERNS_BASE_URL + "StringJoin",
```

Comment:
```suggestion
    link = BUG_PATTERNS_BASE_URL + "StringJoin",
    linkType = CUSTOM,
```
We should define the `linkType` as well, otherwise the URL won't be shown as the [default](https://github.com/google/error-prone/blob/master/annotation/src/main/java/com/google/errorprone/BugPattern.java#L106) is `LinkType.AUTOGENERATED`.

Reply:
Yes, this should be better now, thanks!